stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -111,3 +111,7 @@ spec:
       nodeSelector:
         {{ toYaml .Values.nodeSelector }}
       {{- end }}
+      {{- with .Values.tolerations }}
```

Comment:
Any particular reason to use `with` here instead of `if`?

Reply:
Use `with` instead of `if` can change scope to `.Values.tolerations`. That let me no need to put `.Values.tolerations` to line 116 again, I can use `.` instead.